Tests_v3/Dummy data/MemberPayload.swift

```diff
@@ -7,12 +7,17 @@ import Foundation
 
 extension MemberPayload where ExtraData == NameAndImageExtraData {
     /// Returns a dummy member payload with the given `userId` and `role`
-    static func dummy(userId: UserId = .unique, role: MemberRole = .member) -> MemberPayload {
+    static func dummy(
+        userId: UserId = .unique,
+        createdAt: Date = .unique,
+        updatedAt: Date = .unique,
+        role: MemberRole = .member
+    ) -> MemberPayload {
         .init(
             user: .dummy(userId: userId),
             role: role,
-            createdAt: .unique,
-            updatedAt: .unique
+            createdAt: createdAt,
+            updatedAt: updatedAt
         )
     }
```

Comment:
Do you think we will eventually end up with the same number of parameters as the `init` has and this helper will become pointless? :trollface: